inc/head.php

```diff
@@ -203,7 +209,9 @@ function loadKeyboardFromHash() {
     attachType:'auto',
     setActiveOnRegister:'false'
   }).then(function() {
-    addKeyboards();
+    if(typeof addKeyboards == 'function') {
```

Comment:
This is a separate fix to stop `addKeyboards` from barfing, although it doesn't fix the core problem, which is that if api.keyman.com cannot be reached, then we get no keyboard list 🤣